caseStudies/lib-server.spec.ts

```diff
@@ -0,0 +1,24 @@
+import fs from 'fs';
+import { getTitles, getCaseStudyMd } from './lib-server';
+
+jest.mock('fs');
+
+describe('case studies server-side library', () => {
+  it('read local folder to get markdown file names', () => {
+    fs.readdirSync = jest.fn().mockReturnValue(['caseStudy_1.md', 'otherFile.ts', 'caseStudy_2.md']);
+
+    expect(getTitles()).toEqual(['caseStudy_1', 'caseStudy_2']);
```

Comment:
usually i will reserve getXXX for simpler getters without any usage of adapter (filesystem / api etc)

For this reason, I will prefer `loadTitles` over `getTitles`, and in this case, your test name has good suggestion on what it does, so I will use `loadAllMarkdownFileNames`